requirements.txt

```diff
@@ -3,12 +3,14 @@ amqp==1.4.9
 anyjson==0.3.3
 bcrypt==2.0.0
 billiard==3.3.0.23
+blinker==1.4
```

Comment:
What's this for? I don't see it being used anywhere; it is not a a Flask-Mail dependency